### DISCUSSION TITLE
What is the real-time streaming solution of OpenMLDB?

### DISCUSSION THREAD

**Discussion by QiangCai** [4paradigm/OpenMLDB#1143] [Q&A]
Hi all,
     I have a scenario with real-time streaming loading and online prediction. What is the solution of OpenMLDB? 

Best Regards
David

**Comment by lumianph** [comment]
Hi David, 

Thank you for the interest. Currently OpenMLDB does not explicitly support streaming as data sources. But the development of connectors (such as Kafka connector) is on the roadmap. For a specific scenario, it is not difficult to consume data from streams, you may refer to this demo https://github.com/dl239/OpenMLDB/pull/2
If OpenMLDB is applicable to your applications, we may have a further discussion to see how it can fit into your scenarios. 

Thanks.
